plugins/woocommerce/templates/myaccount/form-edit-account.php

```diff
@@ -44,6 +44,8 @@
 		<input type="email" class="woocommerce-Input woocommerce-Input--email input-text" name="account_email" id="account_email" autocomplete="email" value="<?php echo esc_attr( $user->user_email ); ?>" />
 	</p>
 
+	<?php do_action( 'woocommerce_edit_account_form' ); ?>
```

Comment:
Lint is complaining that this isn't accompanied by a docblock, can we just add a short description of the hook, even if it wasn't there before it might be nice to put one in.